testing/lightdm/APKBUILD

```diff
@@ -1,84 +1,63 @@
 # Contributor: Natanael Copa <ncopa@alpinelinux.org>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=lightdm
-pkgver=1.10.1
-pkgrel=7
+pkgver=1.26.0
+pkgrel=0
 pkgdesc="A cross-desktop display manager"
 url="http://www.freedesktop.org/wiki/Software/LightDM"
 arch="all"
-license="GPL-3.0"
+license="GPL-3.0-or-later"
 depends="dbus xinit accountsservice"
-depends_dev="glib-dev libxklavier-dev libx11-dev"
-makedepends="$depends_dev linux-pam-dev gtk+3.0-dev libxext-dev
+makedepends="linux-pam-dev gtk+3.0-dev libxext-dev libxklavier-dev
 	autoconf automake libtool gobject-introspection-dev itstool
 	libgcrypt-dev libxml2-utils"
 install="$pkgname.pre-install"
 pkgusers="lightdm"
 pkggroups="lightdm"
-subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-source="https://launchpad.net/lightdm/${pkgver%.*}/$pkgver/+download/lightdm-$pkgver.tar.xz
+subpackages="$pkgname-dev $pkgname-doc $pkgname-lang $pkgname-openrc"
+source="https://github.com/CanonicalLtd/${pkgname}/releases/download/${pkgver}/${pkgname}-${pkgver}.tar.xz
 	musl-language.patch
 	musl-is-linux.patch
 	musl-updwtmpx.patch
 	disallow-guest.patch
+	allow-null-passwd.patch
 	lightdm.initd"
 
-_builddir="$srcdir"/lightdm-$pkgver
 prepare() {
-	local i
-	cd "$_builddir"
-	for i in $source; do
-		case $i in
-		*.patch) msg $i; patch -p1 -i "$srcdir"/$i || return 1;;
-		esac
-	done
+	cd "$builddir"
+	default_prepare
 }
```

Comment:
Missed this before, since no special task is done in prepare() the whole function can be deleted and default_prepare will be executed by default :)

Reply:
Ok.